### ISSUE TITLE
nginx -v: Illegal instruction

### ISSUE THREAD

**Issue by WALL-E** [Kong/kong#3071]
### Summary

I download a incorrect RPM from https://getkong.org/install/centos

### Steps To Reproduce

1. download `kong-community-edition-0.11.2.el7.noarch.rpm` from https://getkong.org/install/centos
2. yum install kong-community-edition-0.11.2.el7.noarch.rpm
3. run nginx -v
```
[root@localhost ~]# /usr/local/openresty/nginx/sbin/nginx -v
Illegal instruction
[root@localhost ~]# rpm -qf /usr/local/openresty/nginx/sbin/nginx 
kong-community-edition-0.11.2-1.noarch
```

### Additional Details & Logs

- kong-community-edition.noarch 0:0.11.2-1
- CentOS Linux release 7.2.1511 (Core) 
- VMware  Workstation 11.1.2 build-2780323

**Comment by DanielTimLee** [comment]
well this issue had been closed, 
I think there might be some cpu problems with this **Illegal instruction** problem.

I've been using `Intel Core 2 duo Quad Q6600` (and which doesn't have AVX or AVX2 Instruction)
CentOS Docker images can't execute command of
`kong`
`luarocks`
`nginx`
these. 

I've googled, but there aren't no such references about this, so I'm not sure about this.

`cat /proc/cpuinfo` will show your instructions whether you have AVX instruction or not.
